website/photos/templates/photos/index.html

```diff
@@ -57,7 +57,7 @@
                 <li class="nav-item">
                     <a class="nav-link{% if filter == year|stringformat:"i" %} active{% endif %}"
                        href="{% url 'photos:index-filter' year %}{% urlparams keywords=keys %}#photos-directory">
-                        {{ year }}
+                        {{ year }} / {{ year|add:"1" }}
```

Comment:
```suggestion
                        {{ year }}-{{ year|add:"1" }}
```
we have it like this everywhere else on the website what i see